text/0000-new-browser-support-policy.md

```diff
@@ -0,0 +1,67 @@
+- Start Date: 2020-05-21
+- Relevant Team(s): All
+- RFC PR: (after opening the RFC PR, update this with a link to it and update the file name)
+- Tracking: (leave this empty)
+
+# New Browser Support Policy
+
+## Summary
+
+> In dropping support for IE11 at some point in the future, we will need a new
+browser support policy. This is the proposed policy.
+
+## Motivation
+
+> It should be clear which browsers are supported by Ember.js
+
+## Detailed policy
+
+- Desktop
+
+    1. Google Chrome
+
+        - The current and previous stable releases are supported
+
+    2. Mozilla Firefox
+
+        - The current and previous rapid releases and the latest [ESR](https://support.mozilla.org/en-US/kb/choosing-firefox-update-channel) are supported
+
+    3. MS Edge
+
+        - The current and previous stable releases are supported.
+
+    4. Mac Safari
+
+        - The versions coming with the current and previous releases of MacOS are supported, as well as versions of Mac Safari available with updates to the current release.
+
+- Mobile
+
+    1. Google Chrome
+
+        - The versions coming with all versions of Android released in the past 18 months are supported, as well as the current and most stable releases available via update.
```

Comment:
iOS and Android should have the same release length. I'm not sure why it would be different. I do think it is OK for us to rely on Android users to keep their devices updated so possibly:

> The current and previous version available on any Android device released in the past 36 months.

Reply:
"any Android device" is likely too broad.